frontend/bower.json

```diff
@@ -7,9 +7,10 @@
     "angular-resource": "~1.4.3"
   },
   "dependencies": {
-    "highcharts-ng": "~0.0.10",
+    "highcharts": "~4.2.3",
     "underscore": "~1.8.3",
     "bootstrap-sass": "~3.3.6",
-    "angular-bootstrap": "~0.14.3"
+    "moment": "^2.11.1",
+    "highcharts-more": "0.0.8"
```

Comment:
highcharts-more is also not found by bower. Where are you finding these libraries and how are you adding them to bower?

You should normally do: bower install --save PACKAGE_NAME

That way you insure that bower has found it and they will add it (with the correct version) to bower.json.